packages/domain-picker/package.json

```diff
@@ -47,5 +47,8 @@
 		"@wordpress/i18n": "^3.10.0",
 		"react": "^16.8"
 	},
-	"private": true
+	"private": true,
+	"devDependencies": {
+		"@wordpress/base-styles": "1.9.0"
```

Comment:
It is a devDep because it’s SASS. Will never make it to production. And the lib docs recommended adding it as such.

Reply:
At some point in the monorepo all devDependencies went to root `package.json` — I don't actually know how it is nowadays @Automattic/team-calypso ?

Reply:
You're right when it comes to `client` folder and any sub-project that is not a package. Packages are exempt from this because they're supposed to be atomic and independent of calypso.

Reply:
I was going to raise this point before, but it's a bit irrelevant as long as this is a private package. With the new yarn setup, I believe we no longer have the devDependencies only at the root restriction we did with npm.

That said, this should be a _dependency_ not a _dev dependency_. The package prepare scripts produce assets in the `dist` folder. These assets import sass files which will depend on the package:

https://github.com/Automattic/wp-calypso/blob/406e8a89a531d3925c0fd899d2af6c8228d3a9ce/packages/domain-picker/src/styles/base-styles.scss#L1-L4

This package can't be used without those assets being present. If those assets were bundled into CSS at prepare, this would be a dev dependency, but that's not the case.